modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/leases/LeaseTracker.java

```diff
@@ -92,6 +121,11 @@ public void startTrack() {
                 Lease lease = fromBytes(entry.value());
 
                 leases.put(grpId, lease);
+
+                assert lease.getExpirationTime() != MAX_VALUE : "INFINITE lease expiration time isn't expected";
```

Comment:
I do not understand which is the reason to check expiration time to MAX_VALUE. We never use the constant to define a lease.

Reply:
Removed.